### PR TITLE
feat(portal): implement live code editing with react-live

### PR DESCRIPTION
**Pull request by piofinn**
## 📥 Proposed changes

Implement live code editing in the examples in the portal, using [`react-live`](https://github.com/FormidableLabs/react-live).

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING]("../guides/Contribute.md") doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

Suggested improvements:
* Toggleable dark/light mode
* `overflow-x: scroll` instead of breaking code lines

### CODE REVIEW

File: portal/src/components/Example.tsx [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
+import { LiveProvider, LiveEditor, LiveError, LivePreview } from "react-live";
+import { fremtindLight } from "../utils/utils";
+
 import { Accordion, AccordionItem } from "@fremtind/jkl-accordion-react";
 
 import "./Example.scss";
 
 interface Props {
-    exampleComponent: ReactNode;
+    exampleComponents: {};
```
Comment (lfbergee) [inline portal/src/components/Example.tsx]: Kanskje sette den til å være ReactNode[],

og
```
const scope = exampleComponents.reduce((acc,curr)=>({...acc, curr},{}));
```

eller noe sånt, da beholder vi typesjekken

File: portal/src/pages/components/accordion.tsx [hunk 2]
```diff
@@ -1,32 +1,26 @@
...
+/* Import readme fil with !raw-loader! */
+// @ts-ignore
+import readmeContents from "!raw-loader!@fremtind/jkl-accordion/README.md";
+/* Import code to be rendered as example */
+import { accordionExample } from "../../examples/AccordionExample";
+/* Import components used in the example, and expose them in an object */
 import { Accordion, AccordionItem } from "@fremtind/jkl-accordion-react";
+const components = { Accordion, AccordionItem };
 
 export default function Button() {
```
Comment (lfbergee) [inline portal/src/pages/components/accordion.tsx]: Den henger vel igjen far coppypasta, kunne gjerne fikset den

File: portal/src/pages/components/accordion.tsx [hunk 3]
```diff
@@ -1,32 +1,26 @@
...
-                    <AccordionExample />
-                </div>
-
-                <Accordion>
-                    <AccordionItem title="Kode">
-                        <PrismHighlightedCode>{ButtonExampleCode}</PrismHighlightedCode>
-                    </AccordionItem>
-                </Accordion>
-            </div>
+            <h2 className="jkl-h1 portal-content__title">Trekkspilliste</h2>
```
Comment (lfbergee) [inline portal/src/pages/components/accordion.tsx]: Noen grunn til at vi ikke dogfooder og bruker <H2>?

File: portal/src/pages/components/accordion.tsx [hunk 4]
```diff
@@ -1,32 +1,26 @@
...
+            <Example exampleComponents={components} exampleCode={accordionExample} />
```
Comment (lfbergee) [inline portal/src/pages/components/accordion.tsx]: Kanskje like greit å inline?

`<Example exampleComponents={{Accordion, AccordionItem}} exampleCode={accordionExample} />`
eller, ref andre kommentaren
`<Example exampleComponents={[Accordion, AccordionItem]} exampleCode={accordionExample} />`

File: portal/src/utils/utils.ts [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+        {
+            types: ["builtin", "changed", "keyword"],
+            style: {
+                color: "#ff7059",
+            },
+        },
+        {
+            types: ["number", "inserted"],
+            style: {
+                color: "rgb(181, 206, 168)",
```
Comment (lfbergee) [inline portal/src/utils/utils.ts]: Alle disse rgb fargene er ikke i palleten vår btw. Men så lenge det ser greit ut :)

File: portal/src/pages/components/datepicker.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+/* Import readme fil with !raw-loader! */
+// @ts-ignore
+import readmeContents from "!raw-loader!@fremtind/jkl-datepicker/README.md";
+/* Import code to be rendered as example */
+import { datepickerExample } from "../../examples/DatepickerExample";
+/* Import components used in the example, and expose them in an object */
+import { DatePicker } from "@fremtind/jkl-datepicker-react";
+import "@fremtind/jkl-datepicker/datepicker.min.css";
+
+export default function Button() {
```
Comment (lfbergee) [inline portal/src/pages/components/datepicker.tsx]: button?

File: portal/src/pages/components/datepicker.tsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    return (
+        <Layout>
+            <h2 className="jkl-h1 portal-content__title">Datovelger</h2>
+            <Example exampleComponents={{ DatePicker }} exampleCode={datepickerExample} />
+            <div style={{ margin: "2rem 0" }}>
+                <ReactMarkdown renderers={renderer} source={readmeContents} />
+            </div>
+        </Layout>
```
Comment (lfbergee) [inline portal/src/pages/components/datepicker.tsx]: Missing changelog

File: portal/src/pages/components/accordion.tsx (hunk 3 above)
Comment (piofinn) [reply portal/src/pages/components/accordion.tsx]: Mest for å slippe all importeringa.. Er jo forsåvidt fortsatt dogfooding all den tid vi bruker klassene fra stilpakka ;)

File: portal/src/utils/utils.ts (hunk 5 above)
Comment (piofinn) [reply portal/src/utils/utils.ts]: ✨

**Pull request merged**